express-static/server.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+
+const app = express();
+
+app.use(express.static('public'));
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+// console.log(req.method);
+// console.log(res.send('hello world!'));
+
+// app.use((req, res) => {
+
+// });
```

Comment:
Don't forget to clean up code you don't need.

Reply:
Ah, forgot to get that.  Thx.